lib/sinon/util/core.js

```diff
@@ -167,9 +167,17 @@
                     Object.defineProperty(object, property, wrappedMethodDesc);
                 }
 
```

Comment:
ES5 is sufficient. The 1.17.x branch is in maintenance and any new bigger work should go into the `master` branch, and we won't support pre-ES5 environments in that anyway.